tests/files/Qmin_4000_v2_rkm_with_unknown_key.cfg

```diff
@@ -16,17 +16,17 @@
 [C1]
   Discharge   = 3000.0
   Reference   = reference-Q1_map.nc
-  WithMeasure = reference-Q1_map.nc
+  WithIntervention = reference-Q1_map.nc
 
 [C2]
   Discharge   = 4000.0
   Reference   = reference-Q2_map.nc
-  WithMeasure = reference-Q2_map.nc
+  WithIntervention = reference-Q2_map.nc
 
 [C3]
   Discharge   = 6000.0
   Reference   = reference-Q3_map.nc
-  WithMeasure = measure-Q3_map.nc
+  WithIntervention = measure-Q3_map.nc
```

Comment:
maybe update?

Reply:
update to?

Reply:
check with the test it uses if the update is needed.